main.py

```diff
@@ -12,35 +13,59 @@
 FILL_COLOR = (200, 200, 255)
 entities: List[Entity] = []
 test_entity: Entity
-
-
+sprint: int
+prUpdate: bool
+myfont: pygame.font
+money: int = 0
+distanceX: int
+distanceY: int
 # runs once at the start of the game
 def init():
     global test_entity
+    global money
+    global blob
+    money = 0
+    blob = Entity("blob.png", random.randint(40,1240), random.randint(40,680))
+    entities.append(blob)
     test_entity = Entity("goodsprite.png", 5, 5)
     entities.append(test_entity)
 
-    # create joe and add him to the game like an epic boss
-    joe = Entity("goodsprite.png", 10, 10)
-    entities.append(joe)
-
-
 # runs every frame - game logic
 def update():
     global test_entity
-    print('test entity pos is '+str(test_entity.pos))
-
-    # input stuff, move later to make this less terrible / in the way
+    global blob
+    global money
```

Comment:
just saying when you use `global` you can do like `global test_entity, blob, money`
(at least I think)

Reply:
Will fix